pagopa/io-app
# chore: [IA-577] Display date of birth from profile in place of CF

## Short description
This PR displays the _date of birth_ by considering, as first attempt, the `profile.date_of_birth`
Since it could be undefined, the fallback is the date of birth extracted from the fiscal code

### how to test
- Use the dev-server by changing the [`date_of_birth`](https://github.com/pagopa/io-dev-api-server/blob/88e6dce8a90fd9a445350a62887babfe467155a3/src/payloads/profile.ts#L20) and the [`fiscal_code`](https://github.com/pagopa/io-dev-api-server/blob/88e6dce8a90fd9a445350a62887babfe467155a3/src/config.ts#L27)
- check the birth date in FiscalCodeComponent (profile/tap the fiscal code at the top)
